Lib/test/test_dataclasses.py

```diff
@@ -192,6 +192,20 @@ class C:
         first = next(iter(sig.parameters))
         self.assertEqual('self', first)
 
+    def test_field_named_object(self):
+        @dataclass
+        class C:
+            object: str
+        c=C('foo')
+        self.assertEqual(c.object, 'foo')
+
+    def test_field_named_object_frozen(self):
+        @dataclass(frozen=True)
+        class C:
+            object: str
+        c=C('foo')
```

Comment:
Add spaces around `=`.

Reply:
The only reason I omitted spaces is they are already omitted in `test_field_named_self` which I used as a guideline. Should I fix the `c=C` as well? In this PR or in a separate one?

Reply:
I'd just fix this particular case here, and another PR for all other cases of the same problem.